api/doc/thor.yaml

```diff
@@ -6,6 +6,8 @@ info:
     RESTful API to access VechainThor Network
     
     [Project Home](https://github.com/vechain/thor)
+    
+    ⚠️ <b>Note:</b> The examples given in this specification are optimized for main net. 
```

Comment:
wonder what does make more sense "mainnet" or "main net". Do we have any guidelines ?

Reply:
I would opt for mainnet over main net.